examples/README.md

```diff
@@ -388,8 +388,8 @@ Configure each registry sync:
 				"pollInterval": "6h",               # polling interval, if not set then periodically polling will not run
 				"tlsVerify": true,                  # whether or not to verify tls (default is true)
 				"certDir": "/home/user/certs",      # use certificates at certDir path, if not specified then use the default certs dir
-        "maxRetries": 5,                    # mandatory option! maxRetries in case of temporary errors
-				"retryDelay": "10m",                # mandatory option! delay between retries, retry options are applied for both on demand and periodically sync.
+				"maxRetries": 5,                    # maxRetries in case of temporary errors (default: no retries)
+				"retryDelay": "10m",                # delay between retries, retry options are applied for both on demand and periodically sync.
```

Comment:
we still need the check that if maxRetries is specified then retryDelay must be present.

Reply:
solved, thanks.